plugins/welcome-plugin/resources/welcome-page.css

```diff
@@ -328,10 +328,12 @@ body[class="vscode-light"] .product-logo-dark {
 
 body[class="vscode-light"] .product-logo-light {
     display: block !important;
+    width: 200px;
```

Comment:
I'm afraid it will stretch the image. Could we avoid to use this property?
With what logo have you tested the behavior? Was it default?

Reply:
yep, it changes proportionally

Reply:
It's a welcome page and it's pretty wide. I don't think we have to set a limitation in 200 pixels for images.